example/90-shoot.yaml

```diff
@@ -170,9 +170,8 @@ spec:
   #     - RS256
   #     requiredClaims:
   #       key: value
-  #   authentication: # requires Kubernetes version >= 1.30
-  #     structured:
-  #       configMapName: name-of-configmap-containing-authentication-config
+  #   structuredAuthentication: # requires Kubernetes version >= 1.30
+  #     configMapName: name-of-configmap-containing-authentication-config
```

Comment:
Can we also add a similar change in the `20-garden.yaml` example file: https://github.com/gardener/gardener/blob/master/example/operator/20-garden.yaml